CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ All notable, unreleased changes to this project will be documented in this file.
     - Allow users to enter a list of countries for which no VAT will be added.
 - Fix incorrect payment data for klarna - #7150 by @IKarbowiak
 - Drop deleted images from storage - #7129 by @IKarbowiak
+- Unlink shipping methods from order and checkouts when corresponding ShippingMethodChannelListing is removed - #7164 by @IKarbowiak
```

Comment:
We're fixing an existing feature so no need to document that in the changelog (as we agreed recently on retro).